internal/ssh/ssh.go

```diff
@@ -25,31 +25,16 @@ type Resolver interface {
 	AddrFromModelUUID(ctx context.Context, user openfga.User, modelUUID string) (string, error)
 }
 
-// fowardMessage is the struct holding the information about the jump message received by the ssh client.
-type forwardMessage struct {
-	DestAddr string
-	DestPort uint32
-	SrcAddr  string
-	SrcPort  uint32
-}
-
-// Server is the custom struct to embed the gliderlabs.ssh server and a resolver.
-type Server struct {
-	*ssh.Server
-
-	resolver Resolver
-}
-
 // NewJumpSSHServer creates the jump server struct.
-func NewJumpSSHServer(ctx context.Context, port int, resolver Resolver) (Server, error) {
+func NewJumpSSHServer(ctx context.Context, config SSHServerConfig, resolver Resolver) (Server, error) {
```

Comment:
suggestions:
- rename NewJumpSSHServer to NewJumpServer -> we know it's ssh because it is in the ssh package and it avoids repetition when somebody does `ssh.NewJumpSSHServer` (notice ssh appears twice in this call)
- likewise rename SSHServerConfig to ServerConfig for the same reason